WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -740,4 +784,10 @@ public override void Dispose()
 		}
 		base.Dispose();
 	}
+
+	/// <summary>
+	/// If too many inputs seem to misbehave, problem is probably on coordinator's side.
+	/// Don't ban in that case to avoid huge amount of false-positives.
+	/// </summary>
+	private static bool ReasonableOffendersCount(int offendersCount, int minInputCount) => offendersCount <= minInputCount;
```

Comment:
Guysssss, didn't we want to reduce the banning time (we are not skipping anymore) above a certain percentage?

Right now, if `offendersCount = 140` and `minInputCount = 150`, we will ban 140 coins just as before!